package.json

```diff
@@ -17,8 +17,8 @@
         "knex": "yarn tsn node_modules/.bin/knex --knexfile knexfile.ts",
         "migrate": "yarn knex migrate:latest && yarn typeorm migration:run",
         "lint": "eslint .",
-        "prettify": "yarn prettier --write \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
```

Comment:
Should we maybe leave that one in there if needed, and call it something like `prettify-all`?

Reply:
Added that and ran prettier 2.0 on all files.